app/helpers/devise_helper.rb

```diff
@@ -0,0 +1,15 @@
+module DeviseHelper
+  def devise_error_messages!
+    return '' if resource.errors.empty?
+
+    messages = resource.errors.full_messages.map { |msg| content_tag(:li, msg) }.join
```

Comment:
We feel like this code can't necessarily be easily shortened without adding additional unnecessary lines.